### PR TITLE
Abel/Emmanuel/Jim - Capstone Project

### PR DESCRIPTION
**Pull request by Ntare22**
* Setup kaban board and assign the roles
* Add class including Genre, Author, MusicAlbum etc.
* write test files for various functionalities

### CODE REVIEW

File: classes/genre.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class Genre
+  attr_accessor :items, :genre
+
+  def initialize(name)
+    @name = name
+    @items = []
```
Comment (er-contreras) [inline classes/genre.rb:6]: - [ ] The app is not saving the genres. It shows an empty value when I select the options to list all genres. If you have some specific format please include it in the app.

File: logic/addgame.rb [hunk 2]
```diff
@@ -0,0 +1,69 @@
+require_relative '../classes/game'
+require 'json'
+require 'date'
+
+class AddGame
+  def initialize
+    @games = []
```
Comment (er-contreras) [inline logic/addgame.rb:7]: - [ ] The name of the game it's appearing as null.
Screen Shot 2022-02-17 at 16 21 22

**Review by er-contreras — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

almost there

## Required Changes ♻️
- [ ] The student recorded a video presenting the features of the project.
  - The maximum length of the video is 5 minutes.
  - The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
  - All the members in the video should appear in the video and talk about the project.
  - The video contains a highlight about a piece of the code the student finds interesting or is proud of making.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: classes/genre.rb (hunk 1 above)
Comment (Emmy-github-webdev) [reply classes/genre.rb:6]: Thank you @eri8-9. The bug is fixed

File: logic/addgame.rb (hunk 2 above)
Comment (Emmy-github-webdev) [reply logic/addgame.rb:7]: Thank you @eri8-9. It is fixed

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi Team🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Emmy-github-webdev** [comment]
Thank you @bimbolabuari

**Pull request closed without merging**